enterprise/models.py

```diff
@@ -6,11 +6,14 @@
 
 from uuid import uuid4
 
-from simple_history.models import HistoricalRecords  # likely a bug in import order checker
+import os
+from django.conf import settings
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords  # likely a bug in import order checker
```

Comment:
@mattdrayer @asadiqbal08 I just let it be where `isort` placed it. In case you didn't know that, `isort` without `--check-only` modifies files it checks to put imports in the order it sees as correct.